### DISCUSSION TITLE
Versions Nextclade CLI VS Nextclade Web App

### DISCUSSION THREAD

**Discussion by valentinelsra** [nextstrain/nextclade#665] [Discussion/Chat]
Hello,

Can you confirm that the latest version of Nextclade CLI docker image is the equivalent of the web app version please ?

Thanks,
Valentine.

**Comment by ivan-aksamentov** [comment]
Hi Valentine @valentinelsra,

Sorry for the late reply. I completely missed notifications for this discussion/chat tab. Feel free to open [issues](https://github.com/nextstrain/nextclade/issues) instead - they are more visible for developers, or to join [Nextstrain discussion forum](https://discussion.nextstrain.org/), where you can talk to the community.


Regarding your question, I confirm, and, in fact, we take particular care, that the CLI (docker image or native executable) and Web are kept in sync. The latest version of Web should always use the same algorithms code and produce the same result as CLI (unless there is a bug, in which case please open an [issue](https://github.com/nextstrain/nextclade/issues)). DockerHub container images contain the same exact executable as the corresponding Github Releases.

You can find the mapping between versions of CLI and Web by looking at the [changelog](https://github.com/nextstrain/nextclade/blob/master/CHANGELOG.md). For example this release affects all tools:

```
Nextclade Web 1.13.0, Nextclade CLI 1.10.0, Nextalign CLI 1.10.0 (2022-01-24)
```

And this release:

```
Nextclade CLI 1.10.1 (2022-01-26)
```

affects only Nextclade CLI. In this case, it makes an improvement for command-line flags, the functionality that is not present in Web, so we don't release new version of Web. But the results of CLI 1.10.1 are still the same as of Web 1.13.0.

I hope it makes sense.
